aten/src/ATen/native/transformers/sdp_utils_cpp.h

```diff
@@ -9,4 +13,11 @@ enum class SDPBackend {
   flash_attention = 1,
   efficient_attention = 2
 };
+
+inline c10::SymFloat calculate_scale(const at::Tensor& query, c10::optional<double> scale) {
+    const auto softmax_scale =
+      scale.has_value() ? scale.value() : c10::SymFloat(1) / c10::SymFloat(query.sym_size(-1)).sqrt();
```

Comment:
nit: This heavily relies on consistent interpretation of operator precedence. 
`*(*scale.has_value() ? scale.value() : c10::SymFloat(1)*)* / c10::SymFloat(query.sym_size(-1)).sqrt()`
vs.
`scale.has_value() ? scale.value() : *(*c10::SymFloat(1) / c10::SymFloat(query.sym_size(-1).sqrt()*)*`

We know it does the right thing (hence this is a nit), but I think additional parentheses will make this easier to read and thus use.